symplyphysics/laws/electricity/transmission_lines/ratio_of_average_power_to_incident_power_from_reflection_coefficient.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to_float,
+)
+
+# Description
+## The reflection coefficient is equal to the ratio of the reflected wave power to the incident wave power.
+## Knowing the average power delivered to the load and the incident power, it is possible to calculate the absolute value reflection coefficient.
+
+## Law is: Pa / Pi = 1 - G^2, where
+## Pa - the average power delivered to the load,
+## Pi - the incident power,
+## K - the absolute value reflection coefficient.
```

Comment:
How is it different from normal reflection coefficient?

Reply:
> How is it different from normal reflection coefficient?

This is the reflection coefficient module. That's all the difference.

Reply:
But in this formula it does not matter if it is absolute value or not.

Reply:
Oh, I see. Add some comments to guide future readers on this case.